server/sandbox.go

```diff
@@ -172,6 +172,7 @@ func (s *Server) RunPodSandbox(ctx context.Context, req *pb.RunPodSandboxRequest
 		if err != nil {
 			return nil, err
 		}
+		g.SetProcessSelinuxLabel(processLabel)
 	}
```

Comment:
Do we also need to save the FileLabel, or do you want to regenerate that when we load into the config.json on ocid restart?